scripts/validate_topbraid_uris.py

```diff
@@ -76,16 +122,14 @@ def read_curated_uris(curated_filename):
                 validate_column_header(fields[1], curated_filename)
     return uris
 
-def read_topbraid_uris(topbraid_filename):
+def read_topbraid_uris(topbraid_results):
     uris = {}
-    with open(topbraid_filename) as topbraid_file:
-        reader = csv.DictReader(topbraid_file, dialect='excel-tab',)
-        for row in reader:
-            # subject looks like 'cdd:C001745'
-            uri = row['subject'].split(":")[1]
-            uris[uri] = row['column_header']
-            validate_uri(uri, topbraid_filename)
-            validate_column_header(row['column_header'], topbraid_filename)
+    for clinical_attribute in topbraid_results:
+        # subject looks like 'http://data.mskcc.org/ontologies/clinical_data_dictionary/C002753'
+        uri = clinical_attribute['subject'].split("/")[-1]
+        uris[uri] = clinical_attribute['column_header']
+        validate_uri(uri, "Topbraid")
+        validate_column_header(clinical_attribute['column_header'], "Topbraid")
```

Comment:
I'm not sure why we need to pass in the "Topbraid" argument, since we have only one source. But it is not harmful and we could conceivably have multiple sources in the future (we talked about backup/rollover/override servers)